app/javascript/packs/application.js

```diff
@@ -4,7 +4,7 @@
 // that code so it'll be compiled.
 
 require("@rails/ujs").start()
-require("turbolinks").start()
```

Comment:
I've removed this because I was having strange behaviours with turbolinks. Should we keep it?

Reply:
I don’t think we need this if all the routing is being managed client side by react-router. We could also remove the dependency from gemfile and package.json

Reply:
Yup, that is correct. We don't need that